modules/system/console/WinterInstall.php

```diff
@@ -361,6 +361,8 @@ protected function setupDatabaseSqlsrv()
 
     protected function setupAdminUser()
     {
+        SeedSetupAdmin::$password = Str::random(22);
```

Comment:
You can just edit Line 372 instead to pass `Str::random(22)` as the default value to `$this->ask('Admin Password');`

Reply:
Done.